app/models/order.rb

```diff
@@ -1,6 +1,8 @@
 class Order <ApplicationRecord
   validates_presence_of :name, :address, :city, :state, :zip
+  validates_inclusion_of :status?, :in => [true, false]
```

Comment:
I'm glad we were able to get this to work.

Reply:
Me too! I still have so many questions!